pkg/language/language_test.go

```diff
@@ -161,105 +161,106 @@ func TestWithDetection_Alternate(t *testing.T) {
 
 	assert.Equal(t, heartbeat.LanguageObjectiveC, lang)
 }
 
 func TestDetect_ObjectiveC_M_FileInFolder(t *testing.T) {
-	lang, err := language.Detect("testdata/codefiles/with_mat_file/objective-c.h")
+	lang, err := language.Detect("testdata/codefiles/with_mat_file/objective-c.h", false)
 	require.NoError(t, err)
 
 	assert.Equal(t, heartbeat.LanguageObjectiveC, lang)
 }
 
 func TestDetect_ObjectiveCPP_MatchingHeader(t *testing.T) {
-	lang, err := language.Detect("testdata/codefiles/with_mat_file/objective-cpp.mm")
+	lang, err := language.Detect("testdata/codefiles/with_mat_file/objective-cpp.mm", false)
 	require.NoError(t, err)
 
 	assert.Equal(t, heartbeat.LanguageObjectiveCPP, lang)
 }
 
 func TestDetect_ObjectiveCPP_MM_FileInFolder(t *testing.T) {
-	lang, err := language.Detect("testdata/codefiles/with_mat_file/objective-cpp.h")
+	lang, err := language.Detect("testdata/codefiles/with_mat_file/objective-cpp.h", false)
 	require.NoError(t, err)
 
 	assert.Equal(t, heartbeat.LanguageObjectiveCPP, lang)
 }
 
 func TestDetect_ObjectiveC(t *testing.T) {
-	lang, err := language.Detect("testdata/codefiles/objective-c.m")
+	lang, err := language.Detect("testdata/codefiles/objective-c.m", false)
 	require.NoError(t, err)
 
 	assert.Equal(t, heartbeat.LanguageObjectiveC, lang)
 }
 
 func TestDetect_Matlab_Over_ObjectiveC_Mat_FileInFolder(t *testing.T) {
-	lang, err := language.Detect("testdata/codefiles/with_mat_file/empty.m")
+	lang, err := language.Detect("testdata/codefiles/with_mat_file/empty.m", false)
 	require.NoError(t, err)
 
 	assert.Equal(t, heartbeat.LanguageMatlab, lang)
 }
 
 func TestDetect_ObjectiveC_Over_Matlab_NonMatchingHeader(t *testing.T) {
-	lang, err := language.Detect("testdata/codefiles/matlab_with_headers/empty.m")
+	lang, err := language.Detect("testdata/codefiles/matlab_with_headers/empty.m", false)
 	require.NoError(t, err)
 
 	assert.Equal(t, heartbeat.LanguageObjectiveC, lang)
 }
 
 func TestDetect_NonHeaderFile_C_FilesInFolder(t *testing.T) {
-	lang, err := language.Detect("testdata/codefiles/py_with_c_files/see.py")
+	lang, err := language.Detect("testdata/codefiles/py_with_c_files/see.py", false)
 	require.NoError(t, err)
 
 	assert.Equal(t, heartbeat.LanguagePython, lang)
 }
 
 func TestDetect_Perl_Over_Prolog(t *testing.T) {
-	lang, err := language.Detect("testdata/codefiles/perl.pl")
+	lang, err := language.Detect("testdata/codefiles/perl.pl", false)
 	require.NoError(t, err)
 
 	assert.Equal(t, heartbeat.LanguagePerl, lang)
 }
 
 func TestDetect_FSharp_Over_Forth(t *testing.T) {
-	lang, err := language.Detect("testdata/codefiles/fsharp.fs")
+	lang, err := language.Detect("testdata/codefiles/fsharp.fs", false)
 	require.NoError(t, err)
 
 	assert.Equal(t, heartbeat.LanguageFSharp, lang)
 }
 
 func TestDetect_ChromaTopLanguagesRetrofit(t *testing.T) {
 	tests := map[string]struct {
-		Filepaths []string
-		Expected  heartbeat.Language
+		Filepaths     []string
+		Expected      heartbeat.Language
+		GuessLanguage bool
```

Comment:
Usually `expected` is the last one.
```suggestion
		Filepaths     []string
		GuessLanguage bool
		Expected      heartbeat.Language
```